common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -242,17 +241,17 @@ object CometConf extends ShimCometConf {
       .booleanConf
       .createWithDefault(true)
 
-  val COMET_SHUFFLE_MODE: ConfigEntry[String] = conf(s"$COMET_EXEC_CONFIG_PREFIX.shuffle.mode")
-    .doc("The mode of Comet shuffle. This config is only effective if Comet shuffle " +
-      "is enabled. Available modes are 'native', 'jvm', and 'auto'. " +
-      "'native' is for native shuffle which has best performance in general. " +
-      "'jvm' is for jvm-based columnar shuffle which has higher coverage than native shuffle. " +
-      "'auto' is for Comet to choose the best shuffle mode based on the query plan.")
-    .internal()
-    .stringConf
-    .transform(_.toLowerCase(Locale.ROOT))
-    .checkValues(Set("native", "jvm", "auto"))
-    .createWithDefault("auto")
+  val COMET_NATIVE_SHUFFLE_ENABLED: ConfigEntry[Boolean] =
```

Comment:
What does it mean to enable both native and columnar? The equivalent of 'auto' ?

Reply:
Yes